includes/abstracts/abstract-wc-data.php

```diff
@@ -278,6 +294,14 @@ public function get_meta_data() {
 	 * @return mixed
 	 */
 	public function get_meta( $key = '', $single = true, $context = 'view' ) {
+		if ( ! $this->validate_meta_key( $key ) ) {
+			$function = 'get_' . $key;
```

Comment:
This is our best guess, trying to help them. If they are using `get_meta` where they shouldn't and also their getter is named weirdly, I don't think we can do much